base/slicearray.jl

```diff
@@ -212,7 +212,7 @@ IteratorSize(::Type{Slices{N,L,P,AX,S}}) where {N,L,P,AX,S} = HasShape{N}()
 axes(s::Slices) = s.axes
 size(s::Slices) = map(length, s.axes)
 
-@inline function _slice_index(s::Slices{N,L}, c::Vararg{Int,N}) where {N,L}
+@inline function _slice_index(s::Slices{N,L}, c...) where {N,L}
```

Comment:
You'll need to restrict the `getindex` below (it's _just_ out of reach of the editor here)

Reply:
I'm not sure I understand: what should it be?